minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java

```diff
@@ -624,53 +640,80 @@ private void verifyUp() throws InterruptedException, IOException {
           "Error starting TabletServer " + tsExpectedCount + "- instance not started");
     }
 
-    var zrw = getServerContext().getZooReaderWriter();
-    String rootPath = getServerContext().getZooKeeperRoot();
+    Watcher w = new Watcher() {
+      public void process(WatchedEvent event) {
+        if (event.getState() == KeeperState.Expired) {
+          log.debug("Session expired, state of current session : {}", event.getState());
+        }
+      }
+    };
```

Comment:
I don't think we need to register a watcher. This complicates things.

Reply:
Addressed in 3f7968f